### DISCUSSION TITLE
setting up UGS-CLI to raspberry pi

### DISCUSSION THREAD

**Discussion by Asaad-Sukhon** [winder/Universal-G-Code-Sender#1958] [Help] [answered]
Hello all, 
I was trying to use UGS-CLI on my raspberry pi, I downloaded the repository from here but didn't know how to compile it's code, and didn't find any instructions to walk through. can anyone guide me please. it's kind of urgent.
Thanks in advance.

**Comment by breiler** [comment, marked as answer]
Install Java on your system:
https://github.com/winder/Universal-G-Code-Sender/wiki/Installing#linux-debian-or-ubuntu

Download the UGS classic from here, unpack and run the script `start-cli.sh`:
https://github.com/winder/Universal-G-Code-Sender#downloads

**Reply by Milner99** [reply]
JFrog is asking for login details - is this normal?

**Reply by breiler** [reply]
Darn it...

The downloads are also available from the relases: https://github.com/winder/Universal-G-Code-Sender/releases/tag/v2.0.12
